### PR TITLE
feat(confd): integrated

### PR DESCRIPTION
**Pull request by alebinson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,14 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
 
 In the project directory, you can run:
 
+### `yarn run confd:prod`
```
Comment (shimoncohen) [inline README.md:11]: Please also add documentation about the command:
yarn run confd:prod:indocker

File: src/common/config/index.ts [hunk 2]
```diff
@@ -1,13 +1,17 @@
 /*eslint-disable */
-const MAP_SERVER = 'http://10.28.11.125';
-const PUBLISH_POINT = 'blue_m_flat2d-v001';
-const CHANNEL = 1002;
-const VERSION = 1;
-const REQUEST = 'ImageryMaps';
+const MAP_SERVER = (window as any)._env_.MAP_SERVER;
```
Comment (shimoncohen) [inline src/common/config/index.ts:2]: Multiple user of (window as any)._env_. Maybe assign to a new constant for readability?

Comment (alebinson) [reply src/common/config/index.ts:2]: What do you mean?

File: README.md (hunk 1 above)
Comment (alebinson) [reply README.md:11]: basically this one never will be used directly.
I add it just for us (internally)

Comment (alebinson) [reply README.md:11]: I add new relevant section in **Readme.md**

**Review by shimoncohen — APPROVED** [review]

**Pull request merged**